### PR TITLE
Added AddToQueue(...) in PlayerAPI

### PR DESCRIPTION
**Pull request by lkathke**
I have added a function to add a track to the user's current queue.

### CODE REVIEW

**Comment by DanielLarsenNZ** [comment]
Nice! Thank you so much @lkathke. Would you mind adding an integration test? Here is an example: https://github.com/Ringobot/SpotifyApi.NetCore/blob/master/src/SpotifyApi.NetCore.Tests/PlayerApiTests.cs#L435

File: src/SpotifyApi.NetCore/PlayerApi.cs [hunk 1]
```diff
@@ -778,6 +778,19 @@ public async Task TransferPlayback(string deviceId, bool? play = null, string ac
...
+
+        #endregion
+
+
+        #region AddToQueue
+        public async Task AddToQueue(string itemUri, string accessToken = null, string deviceId = null)
+        {
+            var builder = new UriBuilder($"{BaseUrl}/me/player/queue");
+            builder.AppendToQueryIfValueNotNullOrWhiteSpace("device_id", deviceId);
+            builder.AppendToQueryIfValueNotNullOrWhiteSpace("uri", itemUri);
```
Comment (DanielLarsenNZ) [inline src/SpotifyApi.NetCore/PlayerApi.cs:815]: Please add validation for `itemUri`, e.g.

    if (string.IsNullOrEmpty(itemUri)) throw new ArgumentNullException(nameof(itemUri));

Please put `accessToken` parameter last.

**Review by DanielLarsenNZ — CHANGES_REQUESTED** [review]

File: src/SpotifyApi.NetCore/PlayerApi.cs (hunk 1 above)
Comment (lkathke) [reply src/SpotifyApi.NetCore/PlayerApi.cs:815]: Okay, I have added the variable check and added some test cases

**Comment by DanielLarsenNZ** [comment]
Thanks @lkathke! 👍🏻 Would you like me to publish a release now? Or do you have another PR to include?

**Pull request merged**